factory/config.go

```diff
@@ -37,18 +37,18 @@ const (
 )
 
 type Configuration struct {
-	PcfName         string            `yaml:"pcfName,omitempty"`
-	Sbi             *Sbi              `yaml:"sbi,omitempty"`
-	TimeFormat      string            `yaml:"timeFormat,omitempty"`
-	DefaultBdtRefId string            `yaml:"defaultBdtRefId,omitempty"`
-	NrfUri          string            `yaml:"nrfUri,omitempty"`
-	ServiceList     []Service         `yaml:"serviceList,omitempty"`
-	PlmnList        []PlmnSupportItem `yaml:"plmnList,omitempty"`
-	Mongodb         *Mongodb          `yaml:"mongodb"`
+	PcfName         string    `yaml:"pcfName,omitempty"`
+	Sbi             *Sbi      `yaml:"sbi,omitempty"`
+	TimeFormat      string    `yaml:"timeFormat,omitempty"`
+	DefaultBdtRefId string    `yaml:"defaultBdtRefId,omitempty"`
+	NrfUri          string    `yaml:"nrfUri,omitempty"`
+	ServiceList     []Service `yaml:"serviceList,omitempty"`
+	Mongodb         *Mongodb  `yaml:"mongodb"`
 
 	/* below config received from RoC */
 	DnnList   map[string][]string // sst+sd os key
 	SlicePlmn map[string]PlmnSupportItem
+	PlmnList  []PlmnSupportItem `yaml:"plmnList,omitempty"`
```

Comment:
Is this variable received from ROC? I think it is not. We could do something like below. What do you think?

```diff
diff --git a/factory/config.go b/factory/config.go
index 399ecae..55356ed 100644
--- a/factory/config.go
+++ b/factory/config.go
@@ -46,9 +46,12 @@ type Configuration struct {
        Mongodb         *Mongodb          `yaml:"mongodb"`

-       /* below config received from RoC */
+       // config received from RoC
        DnnList   map[string][]string // sst+sd os key
        SlicePlmn map[string]PlmnSupportItem
+
+       // Add some comment related to this variable
+       PlmnList []PlmnSupportItem `yaml:"plmnList,omitempty"`
 }

 type Service struct {
```

Reply:
done ( I did not know what to put in the comment tho)

Reply:
Ok, no problem. At least we hare separating it from the previous 2 variables and modified their comment. I hope this split/separation is good enough to avoid any confusion. Thanks!